### ISSUE TITLE
"There is news about your certificates" shown with no news

### ISSUE THREAD

**Issue by MikeMcC399** [corona-warn-app/cwa-app-android#4436]
## Avoid duplicates
* [X] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [X] Bug is specific for Android only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [X] Bug is not already reported in another issue
(Related and closed issue for iOS is https://github.com/corona-warn-app/cwa-app-ios/issues/3664.)

## Technical details

- Android version: 11
- App version: 2.14.1

## Describe the bug

When a vaccination certificate is scanned in, the text "There is news about your certificates" with the red dot is always initially shown even if there is no news to be found.

![Example certificate](https://www.coronawarn.app/assets/img/blog/2021-11-04-booster-notifications/news-about-certificates.png)

## Steps to reproduce the issue

1. Tap Certificates
2. Tap Scan
3. Scan in a vaccination certificate
4. Tap back-arrow (top left)
5. Under QR code it says "There is news about your certificates" with a red dot
6. Tap certificate
7. Scroll down to certificate with red dot and tap it
8. View the certificate and look for news, but there is none

## Expected behavior

Only show the text "There is news about your certificates" with the red dot after scanning in a certificate if there is actually some news available associated with the certificate.

---
Internal Tracking ID: [EXPOSUREAPP-10833](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10833)

**Comment by Ein-Tim** [comment]
@MikeMcC399 

I raised the same issue some time ago in the iOS repo: https://github.com/corona-warn-app/cwa-app-ios/issues/3664

@dsarkar Answered in https://github.com/corona-warn-app/cwa-app-ios/issues/3664#issuecomment-939947398: 

"[...] This behaviour works as designed. It was defined, that users who are using another place to scan, will be informed by a red dot and a notification."

However, this issue may be a bit different because you are not scanning "using another place" but are on the certificates tab.

**Comment by MikeMcC399** [comment]
@Ein-Tim 

Thank you for your cross-reference and background to iOS. 

Since the UI for Android is different (compare https://www.coronawarn.app/en/screenshots/#android_certificates to https://www.coronawarn.app/en/screenshots/#ios_certificates) I will mark this issue as for Android only.

**Comment by mtwalli** [comment]
@MikeMcC399  this works as designed. Whenever a certificate is scanned (even from the certificates tab) a badge will be shown to give an indication to whom the certificate belongs. it is very helpful in case many persons are listed in the app.

**Comment by MikeMcC399** [comment]
@mtwalli 
Thank you for your comment.

I revisited this and I now realize that I missed noticing the extra text "New", highlighted in the following screenshot:

![Vaccination new](https://user-images.githubusercontent.com/66998419/144810480-ecfc2723-9843-4cc7-a373-12423fcbe9c7.jpg)

This text is only dismissed after tapping the tile and viewing the certificate details.

![Vaccination](https://user-images.githubusercontent.com/66998419/144810875-ffc660c1-c2ae-4320-904e-8a4667383634.jpg)

I found it confusing that the app tells me that there is news, but then displays nothing which is labelled as news.

I am going to close this issue, marked as bug, if you say it is working as designed. I would however encourage you to think about improving the user experience. To be informed that there is news, but then not to find anything which looks like news is worrying for users in my opinion.
